stories/form.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+
+import Form from '../src/components/Form';
+
+const fieldset =  (
+  <fieldset className="well">
+    <legend>Some input fields</legend>
+    <input placeholder="A field"></input>
+    <input placeholder="Another field"></input>
+  </fieldset>
+);
+const form = storiesOf('Components/Form', module)
+  .add('default', () => {
+    return [
+      <Form
+      title="Default Form"
+      onSubmit={() => {}}
+      content={fieldset}
+      />
+    ]
+  })
+  .add('with messages', () => {
+    return [
+      <Form
+        title="With Info Message"
+        onSubmit={() => {}}
+        content={fieldset}
+        infoText="Use this space to display information about the form!"
+      />,
+      <Form
+        title="With Error Message"
+        onSubmit={() => {}}
+        content={fieldset}
+        errorText="Something went wrong!"
+      />,
+      <Form
+        title="With Success Message"
+        onSubmit={() => {}}
+        content={fieldset}
+        successText="It worked!"
+      />
+    ]
+  })
+  .add('with customized button options', () => {
+    return [
+      <Form
+        title="With Disabled Button"
+        onSubmit={() => {}}
+        content={fieldset}
+        disableButton={true}
+      />,
+      <Form
+        title="With Custom Button Style and Text"
+        onSubmit={() => {}}
+        content={fieldset}
+        buttonClass="centered"
+        buttonContent="This one has a centered button!"
+      />
+    ]
+  })
+  .add('with different styles', () => {
+    return [
+      <Form
+        title="Log In"
+        infoText="Passing in className 'log-in' gives you a centered form with a centered button!"
+        onSubmit={() => {}}
+        content={fieldset}
+        className="log-in"
+      />,
+      <Form
+        onSubmit={() => {}}
+        content={<div className="form-group"><input className="form-control"></input></div>}
```

Comment:
I like how clean this version looks but it seems odd since you are not using the fieldset form you've been using for the rest of the demos. So it'd be good to either point out that the `inline` style needs a specific input (which it shouldn't...) or use the same fieldset example.

Reply:
If you wanted an `inline` form, you would never pass it that fieldset; the whole point of the `inline` style is to have everything on one line, so that would defeat the purpose.  It doesn't need a specific input--it just needs something that will fit on one line.  (It's not as if you need that particular fieldset in order to make the Form component; you can use literally anything as the `content` prop.) . If it's looking confusing, I think the solution would be to stop using the same fieldset in all of the other examples (so as to avoid accidentally giving the impression that something about that particular fieldset is intrinsic to the Form).

Reply:
I would not know to _not_ pass in that fieldset if I wanted an inline form. It's obvious that everything would need to fit on one line but I think it should be stated explicitly. Fieldsets should be used for forms so that should not be removed from the other examples.

Reply:
Fieldsets should be used for forms, but it doesn't have to be THAT fieldset (which is using the styling from the reusable Fieldset component).  I'll make it more explicit!